src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorAdjustedTimeTest_Steps.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+using System.Net;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    public partial class TimeSyncBehaviorAdjustedTimeTest
+    {
+        private void given_an_empty_time_sync_behaviour_state()
+        {
+            this.dateTimeProvider = new DateTimeProvider();
+            var state = new TimeSyncBehaviorState(this.dateTimeProvider, new NodeLifetime(),
+                new AsyncLoopFactory(new LoggerFactory()), new LoggerFactory());
+            this.timesyncBehaviourState = state;
+        }
+
+        private void given_1_outbound_sample()
```

Comment:
This method and the next 2 are basically a copy paste but with different number of repeats. I think it will be a good idea to have just 1 method that accepts and int argument that specifies the number of repeats.

Reply:
agree, I'll update